controllers/bootstrap/init.go

```diff
@@ -457,3 +462,39 @@ func (b *Bootstrap) deleteSubscription(name, namespace string) error {
 
 	return nil
 }
+
+func (b *Bootstrap) waitOperatorReady(name, namespace string) error {
+	if err := utilwait.PollImmediate(time.Second*10, time.Minute*10, func() (done bool, err error) {
+		klog.Info("Waiting for Operator is ready...")
+		key := types.NamespacedName{Name: name, Namespace: namespace}
+		sub := &olmv1alpha1.Subscription{}
+		if err := b.Reader.Get(context.TODO(), key, sub); err != nil {
+			if errors.IsNotFound(err) {
+				klog.V(3).Infof("NotFound subscription %s/%s", namespace, name)
+			} else {
+				klog.Errorf("Failed to get subscription %s/%s", namespace, name)
+			}
+			return false, client.IgnoreNotFound(err)
+		}
+
+		// check csv
+		csvName := sub.Status.InstalledCSV
+		if csvName != "" {
+			csv := &olmv1alpha1.ClusterServiceVersion{}
+			if err := b.Client.Get(context.TODO(), types.NamespacedName{Namespace: namespace, Name: csvName}, csv); err != nil && !errors.IsNotFound(err) {
+				klog.Errorf("Failed to get Cluster Service Version: %v", err)
+				return false, err
+			} else if errors.IsNotFound(err) {
+				klog.Errorf("Notfound Cluster Service Version: %v", err)
+				return false, nil
+			}
```

Comment:
Could we use the same if and else structure as the above line 471-476 for a more clear view?

Reply:
I will update them to 
```
			if err := b.Client.Get(context.TODO(), types.NamespacedName{Namespace: namespace, Name: csvName}, csv); errors.IsNotFound(err) {
				klog.Errorf("Notfound Cluster Service Version: %v", err)
				return false, nil
			} else if err != nil {
				klog.Errorf("Failed to get Cluster Service Version: %v", err)
				return false, err
			}
```

I believe it is clearer.

Reply:
done